addons/mail/static/src/discuss/gif_picker/common/gif_picker.js

```diff
@@ -175,9 +175,16 @@ export class GifPicker extends Component {
             if (this.next) {
                 params.position = this.next;
```

Comment:
And here, this will take the value at the time of the queuing, no at the time of the execute, is that what we want or not?

Reply:
Same question for all other params that are computed outside of the function.

Reply:
> And here, this will take the value at the time of the queuing, no at the time of the execute, is that what we want or not?

I believe we want the original param, not the execute one.
Next contains a string that determine the position to get the "next" gifs. If you get the param at execute time, I think you could miss some next value.
I don't actually think this case can happen since the next value is only relevant when you scroll.